gwinferno/pipeline/parser.py

```diff
@@ -50,11 +58,11 @@ def __init__(self):
     def parse(self, yml_file):
         with open(yml_file, "r") as f:
             yml = yaml.safe_load(f)
-        self.label = yml.pop("label")
-        self.outdir = yml.pop("outdir")
-        self.data_args = yml.pop("data_args")
-        self.sampler_args = yml.pop("sampler_args")
-        self.likelihood_kwargs = yml.pop("likelihood_args")
```

Comment:
It looks like these *are* keyword arguments, right?  Is the idea that the user isn't meant to interact with this, so just think of it as an abstract argument holder?

Reply:
Ahh yeah. I had a lot of these configuration arguments for sampling etc mismatched between named args/kwargs and I tried to set all to the same. 

It would probably make most sense to just label them all kwargs (as I think they all are) since all the yaml parsing puts stuff in dictionaries

This requires just like header changes in the default yamls though to explicitly state they are kwargs. I.e. in https://github.com/FarrOutLab/GWInferno/blob/main/gwinferno/pipeline/config_files/example_config_python_model.yml

Reply:
yeah that makes sense and works just fine -- I can make that change

Reply:
should be all changed with my latest commit